deploy/gunicorn/conf.py

```diff
@@ -12,3 +18,24 @@
 # Configure log structure
 # http://docs.gunicorn.org/en/stable/settings.html#logconfig-dict
 logconfig_dict = logging_config_dict
+
+
+# Because of Gunicorn's pre-fork web server model, we need to initialise opentelemetry
+# in gunicorn's post_fork method in order to instrument our application process, see:
+# https://opentelemetry-python.readthedocs.io/en/latest/examples/fork-process-model/README.html
+def post_fork(server, worker):
+    server.log.info("Worker spawned (pid: %s)", worker.pid)
+
+    resource = Resource.create(attributes={"service.name": "opencodelists"})
+
+    trace.set_tracer_provider(TracerProvider(resource=resource))
+    span_processor = BatchSpanProcessor(
+        OTLPSpanExporter(endpoint="https://api.honeycomb.io")
```

Comment:
this may be duplicated in `OTEL_EXPORTER_OTLP_ENDPOINT` in the environment, I'd like to get it working first & then test removing it. Experimenting with these kind of config tweaks is much easier in production at the moment...!